app/src/components/ArtistEvents.jsx

```diff
@@ -1,32 +1,42 @@
 import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { useConcertData } from "../context/EventContext";
 import useFetch from "../hooks/useFetch";
 import Card from "./Card";
 
-
-
-
 export default function ArtistEvents(props) {
-    const [events, setEvents] = useState([])
-    const id = parseInt(props['id'])
+  const [events, setEvents] = useState([]);
+  const id = parseInt(props["id"]);
 
-    const {
-        error,
-        isPending,
-        data: concerts,
-    } = useFetch("/data/concert_details");
+  const { data: concerts, error, isPending, getEvents } = useConcertData();
+  useEffect(() => {
+    getEvents();
+  }, []);
 
-    useEffect(() => {
-        if (!concerts) return;
-        async function load() {
-            setEvents(concerts.filter(event => event.artist_id === id))
-        }
-        void load()
-    }, [concerts])
+  useEffect(() => {
+    if (!concerts) return;
+    async function load() {
+      setEvents(concerts.filter((event) => event.artist_id === id));
```

Comment:
Ska denna parentesen runt event finnas, har den ett syfte?

Reply:
stylistic choice, its standard lambda syntax which may be enforced by prettier or some other formatter or auto completion tool